Chapter03/recipe-07/cxx-example/menu.yml

```diff
@@ -1,27 +1,22 @@
 appveyor:
   definitions:
-    - CMAKE_CXX_COMPILER: 'g++'
     - Eigen3_DIR: '$HOME/Deps/eigen/share/eigen3/cmake/'
 
 drone:
   definitions:
-    - CMAKE_CXX_COMPILER: 'g++'
     - Eigen3_DIR: '$HOME/Deps/eigen/share/eigen3/cmake/'
 
 travis-linux:
   definitions:
-    - CMAKE_CXX_COMPILER: 'g++'
     - Eigen3_DIR: '$HOME/Deps/eigen/share/eigen3/cmake/'
 
 travis-osx:
   definitions:
-    - CMAKE_CXX_COMPILER: 'clang++'
     - Eigen3_DIR: '$HOME/Deps/eigen/share/eigen3/cmake/'
   failing_generators:
```

Comment:
Why are these expected to fail? I can't remember...

Reply:
I don't know. Should I reactivate and we see what happens?

Reply:
Can't get redder than it already is :smile: I can take over OS X CI business once this one is merged.

Reply:
If we deactivate stuff we should leave comments there in future. I would reactivate everything except what we definitely expect to fail and add some comments.

Reply:
Agreed. Want to do it with this PR?